server.R

```diff
@@ -125,16 +106,6 @@ shinyServer(function(input, output, session) {
     }
   })
 
-  lapply(datatypes, function(x) {
```

Comment:
@rrchai did you mean to drop this block? I don't think I see it in the updated code.

Reply:
yes! This chunk is replaced by scss code in `_header.scss`